src/Xamarin.Android.Build.Tasks/Tasks/BuildApk.cs

```diff
@@ -373,7 +378,13 @@ void AddEnvironment (ZipArchive apk)
 				environment.WriteLine (HttpClientHandlerType == null ? defaultHttpMessageHandler : $"XA_HTTP_CLIENT_HANDLER_TYPE={HttpClientHandlerType.Trim ()}");
 			if (!haveTlsProvider)
 				environment.WriteLine (TlsProvider == null ? defaultTlsProvider : $"XA_TLS_PROVIDER={TlsProvider.Trim ()}");
-			
+			if (!haveMonoGCParams) {
+				if (EnableSGenConcurrent)
+					environment.WriteLine ("MONO_GC_PARAMS=major=marksweep-conc");
+				else
+					environment.WriteLine ("MONO_GC_PARAMS=major=marksweep");
```

Comment:
Is this line necessary? It seems like we could just skip `MONO_GC_PARAMS` emission entirely if `$(AndroidEnableSGenConcurrent)` is False…no?